osu.Game/Rulesets/Difficulty/Preprocessing/DifficultyHitObject.cs

```diff
@@ -25,6 +25,11 @@ public class DifficultyHitObject
         /// </summary>
         public readonly double DeltaTime;
 
+        /// <summary>
+        /// Start time of <see cref="BaseObject"/>.
```

Comment:
```suggestion
        /// The clock-adjusted start time of <see cref="BaseObject"/>.
```

Reply:
Fixed in 010db8968fa664cb7d0a44ec0da727c3a9776773